server.js

```diff
@@ -50,11 +51,24 @@ if (config.auth && config.auth.plugin && config.auth.enabled !== false) {
 
 // Install the Marklogic Provider after installing an authorization plugin.
 const provider = require('./src/koop');
+const https = require("https");
 koop.register(provider);
 
 // Create and configure the Express app server.
 const app = express();
 const port = config.port || 8080;
 app.use('/', koop.server);
-app.listen(port);
-log.info("Koop listening on port " + port + "; press ctrl-C to exit");
+if (!config.ssl.enabled) {
+  app.listen(port);
+  log.info("Koop listening on port " + port + "; press ctrl-C to exit");
+}
+
+if (config.ssl.enabled) {
+  const https = require('https');
+  const options = {
+    key: fs.readFileSync(config.ssl.key, 'utf8'),
+    cert: fs.readFileSync(config.ssl.cert, 'utf8')
+  };
+  https.createServer(options, app).listen(config.ssl.port || 443);
+  log.info(`Koop MarkLogic Provider listening for HTTPS on ${config.ssl.port}`);
```

Comment:
Small nit here - I know this was the previous log message, but I can't think of any reason why it should differ from when SSL is not used. I think `const port = ` above can be changed to `let port = `, and then `port` can be overridden in this block here. And then have the `log.info("Koop listening on port...")` call at the very end (I don't think we need to call out whether it's listening for HTTP or HTTPS traffic, logging the port seems sufficient).